src/include/homestore/btree/detail/varlen_node.hpp

```diff
@@ -510,6 +510,56 @@ class VariableNode : public BtreeNode {
         return str;
     }
 
+    std::string to_string_keys(bool print_friendly = false) const override {
```

Comment:
I don't believe we need this method and we can do it as part of to_string() itself. Having said that since the PR has been pending for long, I think its better to merge this and see if we can fold this into to_string() method itself.